easybuild/easyconfigs/c/Cufflinks/Cufflinks-2.2.1-goolf-1.4.10.eb

```diff
@@ -0,0 +1,29 @@
+name = 'Cufflinks'
+version = '2.2.1'
+
+homepage = 'http://cole-trapnell-lab.github.io/cufflinks/'
+description = """Transcript assembly, differential expression, and differential regulation for RNA-Seq"""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://cole-trapnell-lab.github.io/cufflinks/assets/downloads/']
+
+dependencies = [
+    # issues with boost > 1.55, see https://github.com/cole-trapnell-lab/cufflinks/issues/3
+    ('Boost', '1.51.0', '-Python-2.7.3'),
+    ('SAMtools', '0.1.18'),
+    ('Eigen', '3.1.1'),
```

Comment:
these versions are quite different from the ones in the `foss/2015a` easyconfig?